src/components/FilterBar.jsx

```diff
@@ -17,27 +55,46 @@ export default function FilterBar({ setQueryParam, queryParams }) {
       justifyContent="space-between"
       sx={{ flexGrow: 1 }}>
       <Grid item>
-        <MajorFilter
-          majors={majors}
-          onSelect={(m) => setQueryParam(qParams.MAJORS, m)}
-          onDelete={(m) =>
-            setQueryParam(
-              qParams.MAJORS,
-              majors.filter((major) => major !== m)
-            )
+        <Filter
+          title="Majors"
+          filter={
```

Comment:
Instead of passing a property you can just embed this like normal HTML and read it as the `children` prop.

https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891